packages/react-focus-management/src/FocusManagementProvider.tsx

```diff
@@ -38,17 +41,22 @@ export const useFocusManagementProvider = (
   );
 
   state.dir = state.dir || 'ltr';
-  state.window = state.window || window;
+  state.document = state.document || document;
```

Comment:
We will still need to use there `null`, let's talk about it when we will have SSR tests 🐱